app/api/documents/routes.ts

```diff
@@ -0,0 +1,30 @@
+import { Application, Request, Response, NextFunction } from 'express';
+
+import { validation } from 'api/utils';
+import { documents } from './documents';
+
+export const documentRoutes = (app: Application) => {
+  app.get(
+    '/api/documents/page',
+
+    validation.validateRequest({
+      properties: {
+        query: {
+          properties: {
+            _id: { type: 'string' },
+            page: { type: 'string' },
```

Comment:
what happens if we put here number?

Reply:
you mean pass a page as a number ? as its a query property on the route it is always a string.

Reply:
yes, but doesn't the validator check if that string is a valid number? is it possible?

Reply:
@konz it can be done via regular expression, not needed in my opinion, if you pass something like a not number or a page that does not exist you will get a 404, the main purpose of this validation is to avoid nosql injection, via receiving objects.